dashboard/app/models/levels/free_response.rb

```diff
@@ -72,4 +72,8 @@ def get_property(property_name)
       smart: true
     )
   end
+
+  def validated?
+    true
```

Comment:
I think I commented this on the old PR already, but I don't believe we have any way of validating a free response answer.